packages/medusa/src/api/routes/admin/product-categories/index.ts

```diff
@@ -0,0 +1,17 @@
+import { Router } from "express"
+import middlewares from "../../../middlewares"
+import getProductCategory from './get-product-category'
+
+const route = Router()
+
+export default (app) => {
+  app.use("/product-categories", route)
+
+  route.get("/:id", middlewares.wrap(getProductCategory))
```

Comment:
nit(suggestion, non-blocking): Add `transformQuery` middleware to support `extend` and `fields` in the query when fetching a category. 

I think the entity is small enough that it's this is not essential, just thinking of some of the other endpoints where we now have to retrofit this functionality, @adrien2p?

Reply:
indeed, generally speacking we have to support that. Here it is indeed very small but it is a good practice to have it

Reply:
Got it, didn't know it was for singular requests as well. Thanks for the callout. Added it in.